Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
```

Comment:
I respectfuly disagree. This form of Dockerfile is a result of the mentoring session with Jiří. The container is intended to be used by developers to be able to quickly test their code, and I can confirm that it works as intended. As such, it is not useless.